### PR TITLE
Support rt info attributes which start from numbers

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - XML cannot have attributes which starts from numbers.
 - Pugixml cannot read such IRs
 - The service prefix added for such attributes

### Tickets:
 - 101287

### CODE REVIEW

File: src/core/src/pass/serialize.cpp [hunk 1]
```diff
@@ -795,7 +795,8 @@ void auto_pad_resolving(ov::Node* node) {
 }
 
 void serialize_rt_info(pugi::xml_node& root, const std::string& name, const ov::Any& data) {
-    auto child = root.append_child(name.c_str());
```
Comment (rkazants) [inline src/core/src/pass/serialize.cpp:798]: Before serialization should we check that there is no such attributes starting with the prefix `_ov_prefix_`?

Comment (ilyachur) [reply src/core/src/pass/serialize.cpp:798]: We introduce this prefix ONLY in order to serialize and deserialize tags which start from numbers. It is why we introduce it. So serialized IR will have such prefix.

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**